general-concepts/package-maintainers/text.xml

```diff
@@ -5,6 +5,63 @@
 
 <body>
 
+<section>
+<title>Maintainers</title>
+<body>
+
+<p>
+Package maintainers are responsible for ebuild maintenance tasks —
+bumping them to new upstream releases, updating as necessary wrt
+policies, replying to bugs, etc. They are listed in package's
+<uri link="::ebuild-writing/misc-files/metadata/">metadata.xml</uri>
+file. When assigning bugs, the first maintainer listed becomes the bug's
+assignee and the remaining maintainers are added to CC.
+</p>
+
+<p>
+Package maintainers have authority over their maintained packages.
+Unless specified otherwise, you should request maintainers' approval
+before committing to their packages. The exceptions to this rule are
+trivial changes implied by your own actions, e.g. dependency updates
+as a result of package moves.
+</p>
```

Comment:
This is somewhat redundant to https://devmanual.gentoo.org/ebuild-maintenance/maintenance-tasks/index.html#touching-other-developers-ebuilds. Maybe shorten, and add a cross reference?

Reply:
Hmm, maybe we should move it all here. I feel like 'maintenance tasks' is really a large set of 'this and that', and not really suitable for finding what you're looking for.

Reply:
I went that way. I've added remaining data to maintainers section, split it into two subsections and removed it from 'maintenance tasks'.